### PR TITLE
Simplify/layout

### PR DESCRIPTION
**Pull request by Schwartz10**
Fixes https://github.com/openworklabs/filecoin-web-wallet/issues/170
https://github.com/openworklabs/filecoin-web-wallet/issues/183
https://github.com/openworklabs/filecoin-web-wallet/issues/137

### CODE REVIEW

File: components/Shared/Layout/SidebarLayout.jsx [hunk 1]
```diff
@@ -1,40 +1,24 @@
...
 `
 // Content is a flexible container with no explicit width (hence basis=0) but which grows to consume all available space. It then wraps once its min-width is reached.
 export const Content = styled.div`
   display: flex;
+  justify-content: center;
   flex-basis: 0;
   flex-grow: 999;
-  justify-content: center;
   padding-top: ${props => props.theme.sizes[4]}px;
-  min-width: calc(55% - 1rem);
```
Comment (as-dr) [inline components/Shared/Layout/SidebarLayout.jsx:39]: @Schwartz10 Just wanna point out that removing this means the `SendCard` gets squished a lot more before wrapping. Restoring it doesn't add any problems but allows us to control when, exactly, it wraps. 

Alternatively, to give us more granular control (i.e. to set a different `min-width` for the `SendCard` vs. `TxHistory`, we can institute this in a parent element inside of `Content`.

File: components/Wallet/Send.js/GasCustomization.jsx [hunk 2]
```diff
@@ -8,7 +8,8 @@ import {
   Label,
```
Comment (as-dr) [inline components/Wallet/Send.js/GasCustomization.jsx:8]: Line 109 I get this error:

`Unhandled Rejection (TypeError): Cannot read property 'estimateGas' of null`

```
106 | 
  107 | const [attemptingTx, setAttemptingTx] = useState(false)
  108 | 
> 109 | const estimateGas = async gp => {
      | ^  110 |   // create a fake message
  111 |   const message = new Message({
  112 |     to: wallet.address,
```

Comment (Schwartz10) [reply components/Wallet/Send.js/GasCustomization.jsx:8]: `npm i @openworklabs/filecoin-wallet-provider@latest` try that, i updated the filecoin-wallet-provider

**Review by as-dr — DISMISSED** [review]
One comment on a breaking issue, another on a suggestion (which I'm happy to implement).

File: components/Shared/Layout/SidebarLayout.jsx (hunk 1 above)
Comment (Schwartz10) [reply components/Shared/Layout/SidebarLayout.jsx:39]: I'd say put it back in if you don't see any downside (just make sure no min-width is set somewhere else). I'd say less parent / child elements the better

**Pull request merged**